app/components/autocomplete/at_mention/index.js

```diff
@@ -4,21 +4,43 @@
 import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
-import {getTheme} from 'app/selectors/preferences';
 import {autocompleteUsers} from 'mattermost-redux/actions/users';
-import {getDefaultChannel} from 'mattermost-redux/selectors/entities/channels';
-import {getProfilesInCurrentChannel, getProfilesNotInCurrentChannel} from 'mattermost-redux/selectors/entities/users';
+import {getCurrentChannelId, getDefaultChannel} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+
+import {
+    filterMembersInChannel,
+    filterMembersNotInChannel,
+    filterMembersInCurrentTeam
+} from 'app/selectors/autocomplete';
+import {getTheme} from 'app/selectors/preferences';
 
 import AtMention from './at_mention';
 
+const AT_MENTION_REGEX = /\B(@([^@\r\n\s]*))$/i;
```

Comment:
I think these regexes are being duplicated between this and at_mention.js. Maybe have them exported from at_mention.js and imported here if they're needed in both places?